### DISCUSSION TITLE
搭建archery使用的外部mysql支持8.0吗？

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2270] [General]
搭建archery使用的外部mysql的话，可以使用mysql8.0及以上版本吗？还是说只支持mysql 5.7以上，8.0以下的？

**Comment by LeoQuote** [comment]
应该是支持的, 欢迎测试之后更新 wiki

**Reply by a8159236** [reply]
mysql 8.0测试没问题的，配置没有区别正常使用。

**Reply by LeoQuote** [reply]
谢谢测试, 欢迎更新 wiki
